lib/error.js

```diff
@@ -0,0 +1,20 @@
+import { truncate, inspectList, inspectProperty } from './helpers'
+
+export default function inspectObject(error, options) {
+  const properties = Object.getOwnPropertyNames(error).filter(
+    key => key !== 'stack' && key !== 'line' && key !== 'column' && key !== 'name' && key !== 'message'
```

Comment:
Maybe more readable to define an array of keys and check the existence of the key in it.

```javascript
const nonNativeErrorKeys = [ 'stack', 'line', 'column', 'name', 'message'];
const properties = Object.getOwnPropertyNames(error).filter(key => !(key in nonNativeErrorKeys));
```

Reply:
Maybe using a `Set` is even better?